### ISSUE TITLE
Blender crashes on MacOS 11.1 when this plugin is installed

### ISSUE THREAD

**Issue by JosephHalter** [mrven/Blender-Asset-Creation-Toolset#3]
I'm using the latest version of Blender (2.91.0) and the latest version of Blender Asset Creation Toolset (3.1.2) and when this plugin is installed, Blender crashes frequently with the following error message:

```
Saved session recovery to '/var/folders/rd/kngvlvfs69v5svy8gnzwzw800000gn/T/quit.blend'
Exception in module unregister(): '/Users/joseph/Library/Application Support/Blender/2.91/scripts/addons/Asset_Creation_Toolset_3_1_2_290/__init__.py'
Traceback (most recent call last):
  File "/Applications/Blender.app/Contents/Resources/2.91/scripts/modules/addon_utils.py", line 434, in disable
    mod.unregister()
  File "/Users/joseph/Library/Application Support/Blender/2.91/scripts/addons/Asset_Creation_Toolset_3_1_2_290/__init__.py", line 41, in unregister
    sys.modules[current_module_name].unregister()
  File "/Users/joseph/Library/Application Support/Blender/2.91/scripts/addons/Asset_Creation_Toolset_3_1_2_290/material_tools.py", line 794, in unregister
    bpy.types.CYCLES_PT_context_material.remove(Material_Menu_Panel)
AttributeError: 'RNA_Types' object has no attribute 'CYCLES_PT_context_material'

Blender quit
```

It happens at seemingly random times, when I'm not using the plugin at all, just right clicking on a vertex to select it for example.

**Comment by mrven** [comment]
Do you have enabled Cycles Render Plugin? This required for some functions of ACT.
![Image](https://github.com/mrven/Blender-Asset-Creation-Toolset/assets/6444269/2e95e114-9a85-48ca-8b5f-c115737d99ec)

**Comment by mrven** [comment]
Fix with another bug #4
